pkgs/development/libraries/ffmpegthumbnailer/default.nix

```
@@ -33,7 +33,7 @@ stdenv.mkDerivation rec {
         This thumbnailer was designed to be as fast and lightweight as possible.
         The only dependencies are ffmpeg and libpng.
     ";
-    platforms = platforms.linux;
+    platforms = platforms.all;
```

```suggestion
    platforms = platforms.unix;
```
darwin+linux is unix for meta.platforms.

---

Ah, cool, I will fix that up.